docs/references/databases/create-relationship-attribute.md

```diff
@@ -0,0 +1 @@
+Create relationship attribute.
```

Comment:
```suggestion
Create relationship attribute. [Learn more about relationship attributes](docs/databases-relationships#relationship-attributes).
```

Practically speaking, does this create just the attribute, or does it basically create a relationship?